Docs/sdk/assemblies-for-multiple-vs-versions.md

```diff
@@ -0,0 +1,31 @@
+## Shipping Windows Forms design-time assemblies for multiple Visual Studio versions in a single NuGet package
+
+You may find that you need to ship more than one version of your design-time assemblies, each targeting a different version of Visual Studio.  For example, you might have basic design-time support for your controls starting in VS 17.6, but due to the introduction of some new designer extensibility APIs in 17.7 etc. you might be able to provide a better user experience in 17.8 and later VS versions. In that case you would want to ship two sets of design-time assemblies, one for Visual Studio version 17.6 and the other for 17.8 in a single NuGet package.  
```

Comment:
`VS`->`Visual Studio`  in the whole doc

Reply:
Done for all but 2 that didn't make sense